server/db/seeders/20200131040009-demo-user.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
```

Comment:
If this will be replaced by actual data from the db please add a TODO to remove these files with hardcoded data :)